clients/kafka-common/src/main/java/org/eclipse/hono/client/kafka/consumer/AbstractKafkaConsumerConfigProperties.java

```diff
@@ -16,31 +16,47 @@
 import java.util.HashMap;
 import java.util.Map;
 import java.util.Objects;
+import java.util.Optional;
 
 import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.common.serialization.Deserializer;
 import org.eclipse.hono.client.kafka.AbstractKafkaConfigProperties;
 
 /**
  * Configuration properties for Kafka consumers.
  * <p>
  * This class is intended to be as agnostic to the provided properties as possible in order to be forward-compatible
  * with changes in new versions of the Kafka client.
- *
- * @see <a href="https://kafka.apache.org/documentation/#consumerconfigs">Kafka Consumer Configs</a>
- * @see <a href="https://www.eclipse.org/hono/docs/api/telemetry-kafka">Telemetry API for Kafka Specification</a>
- * @see <a href="https://www.eclipse.org/hono/docs/api/event-kafka">Event API for Kafka Specification</a>
- * @see <a href="https://www.eclipse.org/hono/docs/api/command-and-control-kafka/">Command &amp; Control API for Kafka Specification</a>
  */
-public class KafkaConsumerConfigProperties extends AbstractKafkaConfigProperties {
+public abstract class AbstractKafkaConsumerConfigProperties extends AbstractKafkaConfigProperties {
```

Comment:
I don't quite see why this class needs to be abstract. Couldn't we just keep the existing signature/name?
I think this class should eventually also be used as parameter in the `HonoKafkaConsumer` constructors. And using a non-abstract `KafkaConsumerConfigProperties` class there would be better, IMHO.